deployment/supervisord_initd.sh

```diff
@@ -44,6 +44,12 @@ ulimit -n 96000
 
 RETVAL=0
 
+# source default env vars
```

Comment:
i am not against this. 
but may be just using `[[ ":$PATH:" != */usr/local/bin* ]] && PATH="/usr/local/bin:${PATH}"` here would be sufficient, just an idea